system.h

```diff
@@ -161,5 +165,8 @@ enum {
 #define TIME_ON(tid)  (((tid)==ACTIVE_TIMER)?(TIMING_PORT&=~TIMING_MASK):0)
 #define TIME_TOGGLE(tid)  (((tid)==ACTIVE_TIMER)?(TIMING_PIN|=TIMING_MASK):0)
 
-
+// Voltage Monitoring
+#define VOLTAGE_SENSOR_COUNT 5 // number of sensors we are probing
```

Comment:
comment here isn't quite right.  This is the number of things we're measuring with ADC input pins.  The word 'probe' is reserved for something else in this context